packages/stryker/README.md

```diff
@@ -239,34 +238,37 @@ node_modules directory this would not be possible.
 Stryker will look for the node_modules directory to use in the current basePath (or current working directory) and 
 its parent directories.
 
-#### Plugins  
-**Command line:** `--plugins stryker-html-reporter,stryker-karma-runner`  
-**Config file:** `plugins: ['stryker-html-reporter', 'stryker-karma-runner']`  
-**Default value:** `['stryker-*']`  
-**Mandatory**: no  
-**Description:**  
+### `plugins` [`string[]`]  
+
+Default: `['stryker-*']`  
+Command line: `--plugins stryker-html-reporter,stryker-karma-runner`  
+Config file: `plugins: ['stryker-html-reporter', 'stryker-karma-runner']`  
+ 
+  
 With `plugins` you can add additional Node modules for Stryker to load (or `require`).
 By default, all `node_modules` starting with `stryker-` will be loaded, so you would normally not need to specify this option.
 These modules should be installed right next to stryker. For a current list of plugins,
 you can consult [npm](https://www.npmjs.com/search?q=%40stryker-plugin) or 
 [stryker-mutator.io](https://stryker-mutator.io).
 
-#### Start of port range for test runners
-**Command line:** `--port 9234`  
-**Config file:** `port: 9234`    
-**Default value:** `9234`  
-**Mandatory**: no  
-**Description:**  
+### `port` [`number`]
+
+Default: `9234`  
+Command line: `--port 9234`  
+Config file: `port: 9234`    
+ 
+  
 With `port` you specify the first port to pass on to the test runner to use. Any additional test runners will be spawned using ports n+1, n+2, etc.
 For example, when you set to use port 9234 and Stryker decides to start four test runner processes, ports 9234, 9235, 9236 and 9237 will be passed to the test runner.  
 If the test runner decides to use the port it should be available for use.
 
-#### Global timeout in milliseconds  
-**Command line:** `--timeoutMS 5000`  
-**Config file:** `timeoutMS: 5000`  
-**Default value:** `5000`  
-**Mandatory**: no  
-**Description:**  
+### `timeoutMS` [`number`], `timeoutFactor` [`number`]
```

Comment:
Is it intentional that you have both timeoutMS and timeoutFactor here?

Reply:
That was a mistake, fixed!